client/src/components/AssignPositionModal.js

```diff
@@ -99,8 +99,8 @@ const AssignPositionModal = ({ person, showModal, onCancel, onSuccess }) => {
       const errorMessage = (
         <>
           This position is currently held by{" "}
-          <LinkTo modelType="Person" model={position.person} />. By selecting
-          this position, they will be removed.
+          <LinkToNotPreviewed modelType="Person" model={position.person} />. By
```

Comment:
When clicked on this link,  "Change Position Dialog" is automatically opened in the navigated page. Is this the desired behavior? Also, is it possible to display preview here? Wouldn't it make sense?

Reply:
Hmm, yes it might have been a copy paste mistake, this can and should show preview. Other problem is related to modal state not being closed even if we change person pages. Will fix them both